src/NuGet.Frameworks/NuGetFramework.cs

```diff
@@ -150,11 +150,11 @@ public string DotNetFrameworkName
                         parts.Add(String.Format(CultureInfo.InvariantCulture, "Profile={0}", Profile));
                     }
 
-                    result = String.Join(", ", parts);
+                    result = String.Join(",", parts);
```

Comment:
@emgarten , I am removing the space after `,` to make it consistent with the framework name used by DNX runtime, which is using `System.Runtime.Versioning.FrameworkName`. Is it safe to make this change?